packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1058,6 +1058,7 @@ class _RenderDecoration extends RenderBox {
 
   @override
   double computeDistanceToActualBaseline(TextBaseline baseline) {
+    return computeMaxIntrinsicHeight(0);
     assert(false, 'not implemented');
     return 0.0;
```

Comment:
Any particular reason why the placeholder implementation wasn't removed, or is this just an accident?

Reply:
This is still WIP. There are complexities with textfield baseline calculation, so this is just a temporary line to test with.

Reply:
I am currently planning on fixing this implementation as part of this patch, as inline textfields are an important use of this feature.